cli_test/cli_posts_test.go

```diff
@@ -32,7 +32,7 @@ func TestDesmosCLIPostsCreateNoMediasNoPollData(t *testing.T) {
 	require.Equal(t, startTokens, fooAcc.GetCoins().AmountOf(denom))
 
 	// Create a post
-	success, _, sterr := f.TxPostsCreate(subspace, message, true, fooAddr, "-y")
+	success, _, sterr := f.TxPostsCreate(subspace, message, fooAddr, "-y")
```

Comment:
Can we also add a new test set using `--allows-comments=false` and making sure that the message is generated/sent properly?